src/WebUI/ImageDetails/ImageDetails.tsx

```diff
@@ -48,6 +48,7 @@ export class ImageDetails extends BaseComponent<IImageDetailsProperties, IImageD
     private _getMainHeading(): JSX.Element | null {
         const imageDetails = this.props.imageDetails;
         this._displayImageName = Utils.extractDisplayImageName(imageDetails.imageName);
+        const jobName: string = format("{0} #{1} on {2}", imageDetails.jobName || "", imageDetails.pipelineVersion || "", imageDetails.pipelineName || "");
```

Comment:
this seems to be a link, and also formatlocale, this has to be localized.

Reply:
Added changes to extract url from labels and using that in RunDetails